templates/target/targetExample.md

```diff
@@ -0,0 +1,75 @@
+# Event Target for Splunk
+
+This event target receives arbitrary [CloudEvents][ce] over HTTP and sends them to a [Splunk HTTP Event Collector][hec]
+in a [JSON format][ce-jsonformat].
+
+## Prerequisite
```

Comment:
Just in case someone uses this instead of the template, this should be plural

Reply:
@odacremolbap I agree thank you so much :)